spec.bs

```diff
@@ -253,6 +254,13 @@ Run the [=update document frozenness steps=] given <var ignore>child document</v
 
 Each {{HTMLMediaElement}} has a <dfn for="HTMLMediaElement">resume frozen flag</dfn>, which is initially set to false.
 
+### <a href="https://html.spec.whatwg.org/multipage/web-messaging.html#posting-messages">Posting messages</a> ### {#html-post-message}
+
+Add a note:
+
+NOTE: When posting a message to a {{Window}} that is [=/frozen=], events that are queued on the [=posted message task source=] will not run until the {{Window}} is [=unfrozen=]. If too many
+events are queued the user agent may move the {{Window}} to [=/discarded=] state.
```

Comment:
You can't have a "may" in a note, so we'd want to move this new normative possibility outside of that.

Reply:
Can this say "the user agent *might* move the" ?

Reply:
Is the idea that it would be covered under https://wicg.github.io/page-lifecycle/#discarding ? If so then yeah, that makes sense, although it'd probably be good to elaborate. For example: "might discard the Window's browsing context (as part of the general allowance for discarding under resource pressure)".

Reply:
Sounds awesome, you are such a wordsmith :-)